src/topics/topicActions.js

```diff
@@ -34,3 +35,19 @@ export const fetchTopicsForActiveStream = (narrow: Narrow) => async (
   }
   dispatch(fetchTopics(stream.stream_id));
 };
+
+export const deleteMessagesForTopic = (narrow: Narrow) => async (
+  dispatch: Dispatch,
+  getState: GetState,
+) => {
+  const messages: $ReadOnlyArray<Message | Outbox> = getMessagesForNarrow(getState(), narrow);
```

Comment:
@ray-kraesig can you please help me out here? `getMessagesForNarrow()` is returning an empty array here, even though the narrow does contain several messages.
Output of the console statement: 
![image](https://user-images.githubusercontent.com/7714968/73960907-b2c98d80-4931-11ea-94bc-0b995b939cde.png)

Am I using the selector incorrectly?

Reply:
Posted a discussion on CZO here: https://chat.zulip.org/#narrow/stream/243-mobile-team/topic/.23M3824/near/815558